mflowgen/full_chip/construct.py

```diff
@@ -42,9 +42,11 @@ def construct():
     'flatten_effort'    : 0,
     'topographical'     : True,
     # RTL Generation
-    'array_width'       : 32,
-    'array_height'      : 16,
+    'array_width'       : 12,
+    'array_height'      : 12,
```

Comment:
@steveri @kongty Here is what broke the gold build. We need to change this back to 32x16.

@kongty Was this change intentional?

Reply:
Thanks for finding this @alexcarsello !